pkg/lib/timeformat.ts

```diff
@@ -8,47 +8,60 @@ import { parse, formatDistanceToNow } from 'date-fns';
 import * as locales from 'date-fns/locale';
 
 // this needs to be dynamic, as some pages don't initialize cockpit.language right away
-export const dateFormatLang = () => cockpit.language.replace('_', '-');
+export const dateFormatLang = (): string => cockpit.language.replace('_', '-');
 
-const dateFormatLangDateFns = () => {
+const dateFormatLangDateFns = (): string => {
     if (cockpit.language == "en") return "enUS";
     else return cockpit.language.replace('_', '');
 };
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const dateFnsLocale = (): any => (locales as any)[dateFormatLangDateFns()];
+
+type DTFOptions = {
+    [key: string]: string;
```

Comment:
I think that's spelled `Intl.DateTimeFormatOptions`.

In general it's helpful to go looking inside of `node_modules/typescript/lib/` or `node_modules/@types` to see what this thing is "officially" called, and use that.

Reply:
Thanks! Fixed.